src/storage/DataAccessorBasedStore.ts

```diff
@@ -203,6 +214,9 @@ export class DataAccessorBasedStore implements ResourceStore {
       throw new BadRequestHttpError('Containers should have a `/` at the end of their path, resources should not.');
     }
 
+    if (this.metaStrategy.isAuxiliaryIdentifier(identifier)) {
+      return await this.writeMetadata(identifier, representation);
+    }
     // Ensure the representation is supported by the accessor
     // Containers are not checked because uploaded representations are treated as metadata
     if (!isContainer) {
```

Comment:
Putting it here since the function I expected to be changed wasn't.

I expected the `handleContainerData` function to be updated to make sure the body is empty, instead of making sure it is valid RDF as is being done now. As mentioned in https://github.com/CommunitySolidServer/CommunitySolidServer/issues/1027#issuecomment-1022214820